### ISSUE TITLE
cannot backup firefox

### ISSUE THREAD

**Issue by fajabird** [jensstein/oandbackup#109]
single and batch backup jobs stop at firefox browser forever  - I must kill oandbackup then. Looking at the same time into oandbackup destination folder for firefox I only see some folders but no files in them.

Device:oneplus one
OS: CM12.1 latest stable snapshot

**Comment by jensstein** [comment]
could you send a full logcat with this problem to my email?
is firefox the only app that fails to back up?

**Comment by fajabird** [comment]
yes, leaving out firefox resulted in a complete backup of all remaining
apps (user-apps).
Am Dienstag, den 01.12.2015, 13:28 -0800 schrieb jensstein:

**Comment by jensstein** [comment]
okay, are any of your other user apps comparable to firefox in number of files and size of the app data?

**Comment by fajabird** [comment]
today i wanted to reproduce the issue and send you the logcat - guess what? Yes, cannot reproduce the issue anymore. Could be, that cleaning up firefox offline files (cache etc) helped. Please close this ticket for now. Thank you very much for your prompt reaction.

**Comment by DJaeger** [comment]
@jensstein: If I remember right, this was the reason you wanted to implement a filter for cache files and similar.

**Comment by jensstein** [comment]
yes, a cache filter would be useful here. i haven't looked at it yet though.
